browser/src/Editor/NeovimEditor.tsx

```diff
@@ -361,6 +368,11 @@ export class NeovimEditor implements IEditor {
 
     private _onConfigChanged(): void {
```

Comment:
Update onConfigChanged to pass in the delta configuration (for convenience for subscribes)